UPGRADING.md

```diff
@@ -1,6 +1,20 @@
 Upgrading Grape
 ===============
 
+### Upgrading to >= 1.9.0
+
+#### Response Headers
+
+For Rack >= 3 the following response headers are returned lowercase:
+
+* `content-type`
+* `x-cascade`
+
+For Rack < 3 the following response headers are returned capitalized:
+
+* `Content-Type`
+* `X-Cascade`
+
```

Comment:
Add a link for more information see this PR like below.

I think we need more text explaining what happens in what code. Give an example of an API that sets one of these headers, or explain when they are set?

Reply:
I still want an example here that shows what happens when I was doing `request.heaeders['Content-Type']` before and what I should be doing now. Should I change that code to `request.headers[Grape::Http::Headers::CONTENT_TYPE]` for example? What will the server return before/after?